sdks/wasm/framework/src/WebAssembly.Bindings/LinkDescriptor/WebAssembly.Bindings.xml

```diff
@@ -28,6 +28,7 @@
             <method name="ObjectToString" />
             <method name="GetDateValue" />
             <method name="CreateDateTime" />
+            <method name="CreateUri" />
```

Comment:
Add GetUriType to this list also since you added it along with CreateUri

Reply:
`GetUriType` is marked as public so should not need it.